widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -213,7 +213,7 @@ public static void clearVisitorSession() {
      * Ends active engagement if existing and closes Widgets SDK UI (includes bubble).
      */
     public static void endEngagement() {
-        Logger.d(TAG, "endEngagement");
+        Logger.i(TAG, "End engagement");
```

Comment:
Perhaps using "End Engagement by Integrator" instead would be a more precise description in this context.